pysal/weights/Wsets.py

```diff
@@ -531,3 +531,326 @@ def w_clip(w1, w2, outSP=True, silent_island_warning=False):
     return wc
 
 
+def w_stack(ws, silent_island_warning=False):
```

Comment:
I think we might want to make the `ws` argument spatted?

```python
def w_stack(ws*, silent_island_warning=False): 
```
which would mean that instead of 
```python
>>> w_stack([W1, W2, W3], silent_island_warning=False)
```
we get
```python
>>> w_stack(W1, W2, W3, silent_island_warning=False)
```